evap/evaluation/views.py

```diff
@@ -88,6 +90,8 @@ def index(request):
         )
         return render(request, "index.html", template_data)
 
+    delete_navbar_cache_for_users([request.user])
```

Comment:
I had to dive into the old PR again to find out why exactly this was necessary. Do we maybe want to add a simple comment like:
```python
# The navbar might contain CSRF tokens that will be invalidated by django on login
```
?